Improve portability of easy_pwait in Readme.me (#3)
diazona/pwait: README.md

STOP: Instead you probably want to use something like this

    easy_pwait() {

> 	    while ps -p $1 >/dev/null; do sleep 5; done

Reviewer: whitespace:
```suggestion
            while ps -p $1 >/dev/null; do sleep 5; done
```

>             while ps -p $1 >/dev/null; do sleep 5; done

Reviewer: ```suggestion
        while ps -p $1 >/dev/null; do sleep 5; done
```
Keeping the whitespace consistent between the two functions

    }

> 	

Reviewer: ```suggestion

```
getting rid of trailing whitespace

> or for more portability, which can only check processses owned by the running user

Reviewer: ```suggestion
or a more portable version which can only check processes owned by the running user
```
